src/development/vortex/development/utils/metrics/classification.py

```diff
@@ -824,3 +824,216 @@ def auc(x, y):
     if isinstance(area, np.memmap):
         area = area.dtype.type(area)
     return area
+
+from .metric_base import MetricBase
```

Comment:
I think you can remove all the metric definitions above this, since it is the same as in the `sklearn` implementation.

Reply:
ok

Reply:
updated